back-end/server.js

```diff
@@ -1,13 +1,43 @@
-const express = require('express');
-const cors = require('cors');
+const express = require("express");
+const cors = require("cors");
+const db = require("./data/dbConfig.js");
 
 const server = express();
+const corsOptions = {
+  credentials: true,
+  origin: "https://workout-tracker-pt2.netlify.com/"
+};
 
 server.use(express.json());
-server.use(cors());
+server.use(cors(corsOptions));
 
-server.get('/', (req, res) => {
-  res.send({ message: 'working so far' });
+//custom middleware
+
+function checkForResource(req, res, resource) {
+  if (resource.length > 0) {
```

Comment:
```suggestion
  if (resource.length) {
```
Same thing but you know 🤷‍♂️ other way to do it lol